style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  background: #e5e5e5;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+}
+
+.main {
+  background: url('./images/header-background-shape-desktop.svg');
+  min-height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding-top: 72px;
+}
+
+.toolbar {
+  position: fixed;
+  margin: 0;
+  height: 72px;
+  width: 95vw;
+  padding: 0 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo-wrapper {
+  height: 20px;
+  width: 76px;
+}
```

Comment:
**REQUIRED:** My Logo has much more space on the left-hand side, make it the same with the template, please try to follow the **Figma** as much as possible.

See the difference:
![image](https://user-images.githubusercontent.com/77872355/129120273-75fe96ce-2a2b-46ec-b272-c7736a78e69e.png)
![image](https://user-images.githubusercontent.com/77872355/129120327-10b9a066-5fb3-4074-b897-8a3f0ff456cb.png)